crates/syntax/src/ast/make.rs

```diff
@@ -116,6 +112,10 @@ pub mod ext {
     pub fn ty_result(t: ast::Type, e: ast::Type) -> ast::Type {
         ty_from_text(&format!("Result<{t}, {e}>"))
     }
+
+    pub fn token_tree_from_arg_list(arg_list: ast::ArgList) -> ast::TokenTree {
```

Comment:
We can generalize this I think
```suggestion
    pub fn token_tree_from_node(node: &SyntaxNode) -> ast::TokenTree {
```
we effectively just want to turn any nodes tokens into a `TokenTree` so might as well make this API "untyped" in that regard

Reply:
Nice catch, updated!